ext/src/Decimal.c

```diff
@@ -59,16 +59,18 @@ to_mpf(mpf_t result, php_driver_numeric *decimal)
  * M = mantissa
  */
 #define DOUBLE_MANTISSA_BITS 52
-#define DOUBLE_MANITSSA_MASK (cass_int64_t) ((1LL << DOUBLE_MANTISSA_BITS) - 1)
+#define DOUBLE_MANTISSA_MASK (cass_int64_t) ((1LL << DOUBLE_MANTISSA_BITS) - 1)
 #define DOUBLE_EXPONENT_BITS 11
 #define DOUBLE_EXPONENT_MASK (cass_int64_t) ((1LL << DOUBLE_EXPONENT_BITS) - 1)
 
 static void
 from_double(php_driver_numeric *result, double value)
 {
-  cass_int64_t raw = *((cass_int64_t*) &value);
+  // Copy the bits of value into an int64 so that we can do bit manipulations on it.
+  cass_int64_t raw = 0;
+  memcpy(&raw, &value, 8);
```

Comment:
Doh! The Windows build fails with such an arcane error; thanks a ton. I totally forgot about C89 (or whatever).